src/OperationalInsights/OperationalInsights.Test/ScenarioTests/TableTests.ps1

```diff
@@ -1,5 +1,4 @@
 ﻿# ----------------------------------------------------------------------------------
```

Comment:
please also update src/OperationalInsights/OperationalInsights.Test/ScenarioTests/TableTests.cs if you want to enable new added test cases

Reply:
I added the tests to the .cs file.
before re-recording the tests I want to approve the cmdlets (naming and  final format)